api/v1/routes/tracks.rb

```diff
@@ -3,16 +3,17 @@ module Routes
     class Tracks < Core
       # Status on a track for a user.
       # Called by the CLI.
-      get '/tracks/:id/status' do |id|
+      get '/tracks/:language/status' do |language|
```

Comment:
I've been trying to slowly converge on some terminology for exercism:

- *Track ID* - The url-friendly identifier for a track (e.g. `lfe`)
- *Language* - The name of the programming language (e.g. _Lisp Flavoured Erlang_)

It's not all consistent yet, so there's no reason you should know this. The TL;DR being: let's keep this as `id`, or potentially `track_id` if we want to have more context in the variable name.